src/mol-model-formats/structure/property/bonds/comp.ts

```diff
@@ -56,7 +56,11 @@ export namespace ComponentBond {
         const entries: Map<string, Entry> = new Map();
 
         function addEntry(id: string) {
-            let e = new Entry(id);
+            // weird behavior when 'PRO' is requested - will report a single bond between N and H because a later operation would override real content
```

Comment:
what is this later operation? And why does it override?

Reply:
Protonation variants are processed first. Then content from CCD is written to table. The line `PRO N   H   SING N N 3  ` of the CCD data describes a bond that isn't present in PVCD, so it will be added to the table. This will cause 2 separate occurrences proline (with other comps between them). Downstream operations would only pick the 2nd occurrence, effectively losing all other bond information.